guides/assets/stylesrc/_main.scss

```diff
@@ -17,19 +17,27 @@ body.guide {
 
   -webkit-tap-highlight-color:rgba(38, 27, 35, 0);
 
-  // ----------------------------------------------------------------------------
+  html.translated-rtl & {
```

Comment:
Can we eliminate the usage of `.translated-rtl` from the CSS, and rely on the `dir` alone?

Reply:
Addressed in c0220852